src/plugins/visualizations/public/components/_visualization.scss

```diff
@@ -11,7 +11,7 @@
   flex-direction: column;
   width: 100%;
   height: 100%;
-  overflow: auto;
+  overflow: hidden;
```

Comment:
I think this is too generic and likely needs to be more targeted. Some visualizations require scrolling to be able to access content when resized (e.g. text or controls)